src/main/java/games/strategy/triplea/ui/PlacePanel.java

```diff
@@ -88,9 +88,7 @@ public void territorySelected(final Territory territory, final MouseDetails e) {
       final UnitChooser chooser = new UnitChooser(units, Collections.emptyMap(), getData(),
           false, getMap().getUIContext());
       final String messageText = "Place units in " + territory.getName();
-      if (maxUnits[0] > 0) {
-        chooser.setMaxAndShowMaxButton(maxUnits[0]);
-      }
+      chooser.setMaxAndShowMaxButton(maxUnits[0]);
```

Comment:
Not really following why this change is beneficial. Is there a reason you would want to set a max of 0 or less and show the max unit button?